roboto/bot.py

```diff
@@ -920,6 +922,38 @@ async def stop_poll(
             await make_request(self.session, HTTPMethod.POST, '/stopPoll', request),
         )
 
+    async def send_dice(
+        self,
+        chat_id: Union[ChatID, str],
+        emoji: Optional[str] = ':dice:',
+        disable_notification: Optional[bool] = None,
+        reply_to_message_id: Optional[MessageID] = None,
+        reply_markup: Optional[ReplyMarkup] = None,
+    ) -> Dice:
+        """sendDice API method.
+
+        Args:
+            chat_id: The ID of the chat to send a phone contact to.
+            emoji: A str representation of the emoji of ':dice:' or ':dart:'.
```

Comment:
Avoid mentioning limitations that may be temporary in the docstrings. `:dart:` was added after `:dice:` and there is reason to believe more will be added (the API documentation says "Currently, must be one of “🎲” or “🎯”.").

Notice this is a good place for an Enum: if the bot API is updated and a new emoji is added, we are for sure going to update an Enum, which is real code. We are much less likely to pay attention to updating the docstring, though, which is why I tend to err on the side of not writing such limitations as plain text.

Also, did you test this with a real bot? By reading the API documentation, I'm inclined to believe the user should send the real :game_die: emoji, not `":dice:"`. Maybe both work. An Enum would avoid confusion as well (as long as we are certain of the expected value).